.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -53,9 +53,8 @@ Push access to the upstream repository is required in order to publish the new t
   - [ ] `cargo publish`
 
 - assemble vendor archive:
-  - [ ] `cargo vendor target/vendor`
-  - [ ] `find target/vendor -name '*.a' -delete`
-  - [ ] `tar -czf target/coreos-installer-${RELEASE_VER}-vendor.tar.gz -C target vendor`
+  - [ ] Install https://github.com/cgwalters/cargo-vendor-filterer (at least 0.5.4)
```

Comment:
This should go in the prereqs section so the release engineer doesn't need to check an extra checkbox every time.  Also, maybe it'd be clearer to frame this as a `cargo install` command, similar to the one for `cargo-release`?